ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -910,7 +910,13 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
         ORKTaskProgress progress = [_task progressOfCurrentStep:viewController.step withResult:[self result]];
 
         if (progress.total > 0) {
-            progressLabel = [NSString stringWithFormat:ORKLocalizedString(@"STEP_PROGRESS_FORMAT", nil) ,(unsigned long)progress.current+1, (unsigned long)progress.total];
+            NSString *currentStepLocalizedString = [NSNumberFormatter
+                                                    localizedStringFromNumber:[NSNumber numberWithUnsignedLong:progress.current+1]
+                                                    numberStyle:NSNumberFormatterNoStyle];
```

Comment:
Might be good to create a macro `ORKLocalizedStringFromNumber()` in `ORKDefines_Private.h`, so we can make these code cleaner.